packages/web/src/scss/components/Tooltip/index.html

```diff
@@ -575,6 +664,7 @@ <h2 class="docs-Heading">Advanced Floating Functionality</h2>
               data-spirit-enable-shifting="true"
               data-spirit-enable-sizing="true"
               data-spirit-placement="top-start"
+              data-spirit-enable-hover="true"
```

Comment:
Is this needed when we have the `triggers`?

Reply:
removed